File: java/folder program/Gadget.java
public class Gadget extends ProdukElektronik {
    private String jenis;
    private String garansi;
    private String status;
    private String stok;

    public Gadget() {}

    public Gadget(String kode,String nama, int harga, String deskripsi,
                  String jenis, String garansi, String status, String stok) {
        super(kode,nama, harga, deskripsi);
        this.jenis = jenis;
        this.garansi = garansi;
        this.status = status;
        this.stok = stok;
    }

    // Getter & Setter
    public String getJenis() { return jenis; }
    public void setJenis(String jenis) { this.jenis = jenis; }

    public String getGaransi() { return garansi; }
    public void setGaransi(String garansi) { this.garansi = garansi; }

    public String getStatus() { return status; }
    public void setStatus(String status) { this.status = status; }

    public String getStok() { return stok; }
    public void setStok(String stok) { this.stok = stok; }

    @Override
    public void displayInfo() {
        super.displayInfo();
        System.out.println("Jenis: " + jenis);
        System.out.println("Garansi: " + garansi);
        System.out.println("Status: " + status);
        System.out.println("Stok: " + stok);
    }
}

File: cpp/folder program/Gadget.cpp
#include <iostream>
#include "ProdukElektronik.cpp"

using namespace std;

class Gadget : public ProdukElektronik{
    private:

    string jenis;
    string garansi;
    string status;
    string stok;

    public:

    Gadget(){

    }

    void setjenis(string jenis){
        this -> jenis=jenis;
    }
    string getjenis(){
        return this->jenis;
    }

    void setgaransi(string garansi){
        this->garansi=garansi;
    }
    string getgaransi(){
        return this->garansi;
    }
    
    void setstatus(string status){
        this->status=status;
    }
    string getstatus(){
        return this->status;
    }

    void setstok(string stok){
        this->stok=stok;
    }
    string getstok(){
        return this->stok;
    }


};

// getter setter


